articles/azure/azs-home.md

```diff
@@ -0,0 +1,70 @@
+---
+title: UKCloud for Microsoft Azure | UKCloud Ltd
+description: Home page for UKCloud for Microsoft Azure
+services: azure-stack
+author: Sue Highmoor
+reviewer:
+lastreviewed: 
+toc_rootlink: About
+toc_sub1: 
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: UKCloud for Microsoft Azure
+toc_fullpath: About/azs-home.md
+toc_mdlink: azs-home.md
+---
+
+# UKCloud for Microsoft Azure
+
+UKCloud for Microsoft Azure harnesses the innovation of Microsoft’s Azure platform delivered from within our award-winning government-grade multi-cloud environment – addressing the service, connectivity and diversity requirements of UK public sector organisations and their industry partners.
+
+## Let's get started
+
+First, take a look at our [*Getting Started Guide*](azs-gs.md) to learn the basics, then you can:
+
+<div class="row">
+  <div class="col-md-3"><ul><li><a href="https://docs.ukcloud.com/articles/azure/azs-how-create-vm.html">Create a VM (using the Portal)</a></ul></div>
+  <div class="col-md-3"><ul><li><a href="https://docs.ukcloud.com/articles/azure/azs-how-create-vm-cli.html">Create a VM (using the Azure CLI)</a></ul></div>
+  <div class="col-md-3"><ul><li><a href="https://docs.ukcloud.com/articles/azure/azs-how-manage-azure-group.html">Manage groups and permissions</a></ul></div>
+  <div class="col-md-3"><ul><li><a href="https://docs.ukcloud.com/articles/azure/azs-how-retrieve-quota-ps.html">Retrieve your subscription quotas using PowerShell</a></ul></div>
+</div>
+
+<div class="row">
+  <div class="col-md-3"><ul><li><a href="https://docs.ukcloud.com/articles/azure/azs-how-create-vm-ps.html">Create a VM (using PowerShell)</a></ul></div>
+  <div class="col-md-3"><ul><li><a href="https://docs.ukcloud.com/articles/azure/azs-how-create-vm-terraform.html">Create a VM (using Terraform)</a></ul></div>
+  <div class="col-md-3"><ul><li><a href="https://docs.ukcloud.com/articles/azure/azs-how-configure-powershell-users.html">Configure PowerShell environment</a></ul></div>
+  <div class="col-md-3"><ul><li><a href="https://docs.ukcloud.com/articles/azure/azs-how-use-azure-api.html">Use the Azure Stack API</a></ul></div>
+</div>
+
+## Azure Stack portal overview
+
+<div class="row">
+  <div class="col-md-6">
+    <div style="padding:56.25% 0 0 0;position:relative;"><iframe src="https://player.vimeo.com/video/305064724?color=ffffff&title=0&byline=0&portrait=0" style="position:absolute;top:0;left:0;width:100%;height:100%;" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe></div><script src="https://player.vimeo.com/api/player.js"></script>
```

Comment:
Could the styles in here be extracted out to the CSS file, keep it a bit tidier
Could newlines be added between elements too to aid readability

Reply:
Added line breaks